### PR TITLE
Dev/gbiw 141 tslint

### PR DESCRIPTION
**Pull request by matthijsln**
I did not run `npm run lint-fix` yet, because of GBIW-140.

### CODE REVIEW

File: viewer/src/main/webapp/flamingo-components/package.json [hunk 1]
```diff
@@ -7,7 +7,8 @@
     "build": "ng build",
     "build-prod": "npm run ng -- build --prod --output-hashing=none",
     "test": "ng test --project=core",
-    "lint": "ng lint",
+    "lint": "ng lint --project=bridge",
+    "lint-fix": "ng lint --project=bridge --fix",
```
Comment (mtoonen) [inline viewer/src/main/webapp/flamingo-components/package.json:11]: This should be project=bridge, but also core
bridge is the glue between angular and extjs, core contains the actual components

**Comment by mtoonen** [comment]
Please use feat/GBIW-123_ for branchnames. This will make youtrack pick up commits

File: viewer/src/main/webapp/flamingo-components/package.json (hunk 1 above)
Comment (matthijsln) [reply viewer/src/main/webapp/flamingo-components/package.json:11]: bridge project also lints core files

**Comment by matthijsln** [comment]
Superseded by #1973 1973

**Pull request closed without merging**